cli_tools/gce_ovf_import/ovf_importer/ovf_importer_test.go

```diff
@@ -182,6 +183,50 @@ func TestSetUpWorkflowHappyPathFromOVAExistingScratchBucketProjectZoneAsFlags(t
 	assert.Equal(t, "gs://bucket/folder/ovf-import-build123/ovf/", oi.gcsPathToClean)
 }
 
+func TestSetUpWorkflowUsesImageLocationForBetaReleaseTrack(t *testing.T) {
```

Comment:
How about for GA? Do you want to assert that the location is not set?

Reply:
Added a test case for GA